MAC_SETUP.md

```diff
@@ -127,36 +123,15 @@ source venv-name/bin/activate
 
 Typically you will name the venv `venv`, or `venv38` for a python 3.8 venv.
 
+## Nox
```

Comment:
This info is in contributing. I'd prefer we don't repeat this info, but just include the mac specific information and link to the contributing doc section

Reply:
I didn't see this info in contributing? Maybe I am just overlooking it?

Reply:
It's in the Authoring Guide, which is linked to from CONTRIBUTING.

Reply:
I'd also like to add a link to the first mention of nox a few lines above, since this doc is the first time I ever heard of the tool and I initially thought it was a typo for "now".